core/domain/email_manager.py

```diff
@@ -2045,18 +2047,20 @@ def send_mail_to_notify_contributor_ranking_achievement(
 
 def send_reminder_mail_to_notify_curriculum_admins(
     curriculum_admin_ids: List[str],
-    overdue_stories_dicts: List[Dict[str, Sequence[str]]],
-    upcoming_stories_dicts: List[Dict[str, Sequence[str]]]) -> None:
+    overdue_stories_dicts: List[story_domain.OverdueStoryDict],
```

Comment:
Use domain objects, not dicts, when passing data around the domain layer.

Reply:
Done